openml-utils/src/main/java/com/feedzai/openml/util/validate/ClassificationValidationUtils.java

```diff
@@ -70,4 +76,33 @@ public static void validateClassificationModel(final DatasetSchema schema, final
             throw new ModelLoadingException(msg, e);
         }
     }
+
+    /**
+     * Validates that the model to load can be used with the given parameters.
+     *
+     * @param modelLoader An instance responsible for instantiating models.
+     * @param modelPath   Path of the model.
+     * @param schema      Schema of the model.
+     * @param params      Collection of parameters and the corresponding values.
+     * @throws ModelLoadingException If the path and/or schema are not valid.
+     * @since @@@feedzai.next.release@@@
```

Comment:
We do not have this automated here AFAIK, so you probably need to put the since manually

Reply:
Should we open a feature request (i.e. issue) to automate this?